src/Status/StatusCard/StatusCard.stories.tsx

```diff
@@ -44,3 +105,14 @@ export const WithIconTooltip = {
 
   render: Template
 };
+
+/**
+ * This story will display a different title variant and truncated title with tooltip if it is too long
+ */
+export const WithGridLayout = {
```

Comment:
We should disable the controls that are not relevant for this story. e.g. titleVariant seems to be the only that actually changes anything.